Lib/_compat_pickle.py

```diff
@@ -87,6 +87,7 @@
     ('urllib', 'urlretrieve'): ('urllib.request', 'urlretrieve'),
     ('urllib2', 'HTTPError'): ('urllib.error', 'HTTPError'),
     ('urllib2', 'URLError'): ('urllib.error', 'URLError'),
+    ('exceptions', 'IncompleteInputError'): ('builtins', 'IncompleteInputError'),
```

Comment:
Was there such exception in Python 2? :confused:

Reply:
This is `NAME_MAPPING` that maps general exceptions to the module, the python 2 exceptions are below. I updated this because there is a test that is expecting all builtin exceptions to be available in this mapping so that made me think this mapping should be up to date. I am missing something?

Reply:
This is a mapping that maps Python 2 names to Python 3 names. It allows to load in Python 3 pickles created in Python 2. If the name did not exist in Python 2, it will not be in this mapping. For example you will not find numerous OSError subclasses here. Multiple names can be mapped to the same name.

The reverse mapping helps to create pickles compatible with Python 2. OSError subclasses are mapped to OSError. The ImportError subclass is mapped to ImportError.

So, you can add a mapping of IncompleteInputError to SyntaxError in REVERSE_NAME_MAPPING if you want to produce Python 2 compatible pickles. But I do not know why you can want to do this.

Or you can make the test to skip IncompleteInputError. Oh, there is already a pretty long list of skipped exceptions, from BlockingIOError to ExceptionGroup.

Oh, actually I wrote these tests. Initially the list of the skipped exceptions consisted only from two elements. I would extract that list in a global variable.

Reply:
Ah, thanks for the context. I added the skip to test_exceptions to match how we have been treating new ones recently.